### ISSUE TITLE
Vignette Documentation: Language on how step_zv works?

### ISSUE THREAD

**Issue by mpettis** [tidymodels/tidymodels.org-legacy#185]
I have been reading https://www.tidymodels.org/start/recipes/, regarding zero-value predictors, and I came across this sentence:

> When the recipe is applied to the training set, a column is made for LEX but it will contain all zeros.

From my read of the code, since the recipe references the training set, that doesn't have the LEX value in column `dest`.  I don't understand why it would create a zero-value dummy variable in that case, since it was never present in the training data referenced.  I can see the need that, if we make the dummies off of the training set, apply the same recipe to the test set, which will generate a LEX column, that we want to tell the model fit process "don't use the LEX column in the test set, as it was never trained on in the training set."

Could someone clear this up for me?  Either the verbiage is wrong, or I have a fundamental misconception about what is going on in recipes with the zero-value predictor.

**Comment by juliasilge** [comment]
I can understand how this is confusing! We should clear it up.

This is happening because `dest` is a **factor**:

```
> class(train_data$dest)
[1] "factor"
> levels(train_data$dest)
  [1] "ABQ" "ACK" "ALB" "ANC" "ATL" "AUS" "AVL" "BDL" "BGR" "BHM" "BNA" "BOS"
 [13] "BQN" "BTV" "BUF" "BUR" "BWI" "BZN" "CAE" "CAK" "CHO" "CHS" "CLE" "CLT"
 [25] "CMH" "CRW" "CVG" "DAY" "DCA" "DEN" "DFW" "DSM" "DTW" "EGE" "EYW" "FLL"
 [37] "GRR" "GSO" "GSP" "HDN" "HNL" "HOU" "IAD" "IAH" "ILM" "IND" "JAC" "JAX"
 [49] "LAS" "LAX" "LEX" "LGB" "MCI" "MCO" "MDW" "MEM" "MHT" "MIA" "MKE" "MSN"
 [61] "MSP" "MSY" "MTJ" "MVY" "MYR" "OAK" "OKC" "OMA" "ORD" "ORF" "PBI" "PDX"
 [73] "PHL" "PHX" "PIT" "PSE" "PSP" "PVD" "PWM" "RDU" "RIC" "ROC" "RSW" "SAN"
 [85] "SAT" "SAV" "SBN" "SDF" "SEA" "SFO" "SJC" "SJU" "SLC" "SMF" "SNA" "SRQ"
 [97] "STL" "STT" "SYR" "TPA" "TUL" "TVC" "TYS" "XNA"
```

LEX is there as a factor level for the training data (the factor levels came from before the splitting) even though there are no examples of LEX in the training data. We should add another sentence to make this more clear.

**Comment by mpettis** [comment]
Got it, thanks.  The critical part you wrote was that the factors were created before the split, and so would become a dummy.  A closer reading by me would have cleared it up, but you are right, noting that at this location in the vignette would help people avoid that misconception.
